spec/features/course/assessment/question/programming_management_spec.rb

```diff
@@ -151,7 +151,7 @@
         page.find('#programming-question-form-submit').click
         wait_for_job
 
-        # expect(page).to have_no_xpath('//form//*[contains(@class, \'fa-spinner\')]')
+        expect(page).to have_no_xpath('//form//*[contains(@class, \'fa-spinner\')]')
```

Comment:
if you have removed all fa icons, this test line is meaningless since fa will never exist anymore. You should change this to the new icon

Reply:
do you think it's safe to just delete this test?

Reply:
Definitely not. By the way the same code also exists in other lines. the purpose of this code is to test that the icon is not there.. I think it was commented previously since this particular test is flaky but other tests with same code still work

Reply:
I see okay noted on that!! I will work on all the rspec that involves this

Reply:
Done